pact/coin-contract/coin.pact

```diff
@@ -147,12 +165,24 @@
     \chain. This fails if either SENDER or RECEIVER does not exist.           \
     \Create-on-transfer can be done using the 'transfer-and-create' function."
 
+    @model [
+      (property conserves-mass)
+      (property (> amount 0.0))
+      (property (not (= sender "")))
+      (property (not (= receiver "")))
+    ]
+
     (enforce (not (= sender receiver))
       "sender cannot be the receiver of a transfer")
 
+    (enforce (!= "" sender) "empty sender")
+    (enforce (!= "" receiver) "empty sender")
```

Comment:
This wasn't guarded against elsewhere?

Reply:
@fosskers No - since this costs gas, the sender of the tx is incentivized to not grief us by sending in empty values.

Reply:
Also FV was busted and we did have some models for this, which once FV was fixed, failed ...